database/migrations/2019_06_21_180451_create_sym_links_table.php

```diff
@@ -14,6 +14,7 @@ class CreateSymLinksTable extends Migration
 	 */
 	public function up()
 	{
+		Schema::dropIfExists('sym_links');
```

Comment:
I forget - Does the code clear up orphaned symlinks, or will this leave some around?

Reply:
hummm. Indeed. But that would just be emptying the `public/sym/` directory.

Reply:
True. Could add a note to the log, but there are likely to be enough symlinks in that folder that no-one's really likely to notice.